buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -827,7 +828,7 @@ class BeamModulePlugin implements Plugin<Project> {
         options.compilerArgs += ([
           '-parameters',
           '-Xlint:all',
-          '-Werror'
+          //'-Werror'
```

Comment:
Do we want to remove this?

Reply:
No this wasn't intentional, thank you. I just rebased to resolve merge conflicts and also addressed this.